src/App.tsx

```diff
@@ -1,54 +1,95 @@
-import React from 'react';
-import './App.scss';
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
+
 import { peopleFromServer } from './data/people';
 
+import './App.scss';
+// import { Person } from './types/Person';
```

Comment:
Remove comment. Try to write obvious code with meaningful variable and function naming